packages/pipeline/src/data_sources/trusted_tokens/index.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+export interface ZeroExTrustedTokenMeta {
+    address: string;
+    name: string;
+    symbol: string;
+    decimals: number;
+}
+
+export interface MetamaskTrustedTokenMeta {
+    address: string;
+    name: string;
+    erc20: boolean;
+    symbol: string;
+    decimals: number;
+}
+
+export class TrustedTokenSource<T> {
```

Comment:
nit: Since the type parameter `T` is only used in the `getTrustedTokenMetaAsync` method, we could declare it there instead of at the class level.